acos_client.egg-info/PKG-INFO

```diff
@@ -0,0 +1,109 @@
+Metadata-Version: 1.1
```

Comment:
Had you left the .gitignore in place, these would have been automatically ignored. :)